### DISCUSSION TITLE
Setting up harvester - error: socket.gaierror: [Errno 11001] getaddrinfo failed

### DISCUSSION THREAD

**Discussion by databursted** [Chia-Network/chia-blockchain#5829] [Support]
Hello support, I have a full node set up performing farming duties and also plotting (both Windows 10 machines).  I am adding a harvester machine to help with plotting.  I have followed the steps to set that up per:  
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

However when I get to step #6, and run command: chia start harvester -r   I get the following messages:
C:\Users\MyName\AppData\Local\chia-blockchain\app-1.1.6\resources\app.asar.unpacked\daemon>chia start harvester -r
Daemon not started yet
Starting daemon
chia_harvester: started

C:\Users\MyName\AppData\Local\chia-blockchain\app-1.1.6\resources\app.asar.unpacked\daemon>Traceback (most recent call last):
  File "chia\server\start_harvester.py", line 57, in <module>
  File "chia\server\start_harvester.py", line 53, in main
  File "chia\server\start_service.py", line 226, in run_service
  File "asyncio\runners.py", line 43, in run
  File "asyncio\base_events.py", line 587, in run_until_complete
  File "chia\server\start_service.py", line 220, in async_run_service
  File "chia\server\start_service.py", line 160, in run
  File "chia\server\start_service.py", line 139, in start
  File "chia\server\start_service.py", line 139, in <listcomp>
  File "chia\server\reconnect_task.py", line 12, in start_reconnect_task
socket.gaierror: [Errno 11001] getaddrinfo failed
[12592] Failed to execute script start_harvester

I tried it on a machine had a full node install on this machine previously and deleted it as much as I could. But I also placed a Windows 10 image on the machine which has never seen Chia up until this point and following the same steps I get the same error.  Any suggestions?

**Comment by BrandtH22** [comment]
@databursted , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
